Source/CBL_BlobStore+Internal.h

```diff
@@ -0,0 +1,25 @@
+//
+//  CBL_BlobStore+Internal.h
+//  CouchbaseLite
+//
+//  Created by Jens Alfke on 8/23/15.
+//  Copyright © 2015 Couchbase, Inc. All rights reserved.
+//
+
+#import "CBL_BlobStore.h"
+
+
+// Name of file in blob-store dir that records encryption type used (currently "AES")
+#define kEncryptionMarkerFilename @"_encryption"
+
+
+@interface CBL_BlobStore ()
+
+- (NSString*) rawPathForKey: (CBLBlobKey)key;
+@property (readonly, nonatomic) NSString* tempDir;
+@property (readonly) CBLSymmetricKey* encryptionKey;
+
+#if DEBUG
+- (BOOL) markEncrypted: (BOOL)encrypted error: (NSError**)outError; // exposed for testing ONLY
+#endif
```

Comment:
It's a private method of CBL_BlobStore. _Outside_ that class, it should only be used for testing. It's only the declaration of the method in a header that's ifdef'd.